.github/workflows/lint.yml

```diff
@@ -10,4 +10,4 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v2
         with:
-          version: v1.44.0
+          version: v1.44.2
```

Comment:
the version v1.45.0 is out https://github.com/golangci/golangci-lint/releases/tag/v1.45.0 can you update that?

Reply:
Done.